### ISSUE TITLE
Import error in Django 3.1

### ISSUE THREAD

**Issue by jleclanche** [ivelum/djangoql#57]
Compatibility issue with Django 3.1b1.
```pytb
  File ".../lib/python3.8/site-packages/djangoql/schema.py", line 8, in <module>
    from django.db.models import FieldDoesNotExist, ManyToManyRel, ManyToOneRel
ImportError: cannot import name 'FieldDoesNotExist' from 'django.db.models' (.../lib/python3.8/site-packages/django/db/models/__init__.py)
```

**Comment by iceone** [comment]
Hi, that have been fixed in 0.14.0

**Comment by jleclanche** [comment]
Thanks! :)

PS: I see you're struggling with the setup.py and readme and all… I recommend [Poetry](https://python-poetry.org/). It's very stable now and every library should use it, makes everything a lot simpler :)
